### ISSUE TITLE
My plan only builds default Iosevka

### ISSUE THREAD

**Issue by arudp** [be5invis/Iosevka#761]
It seems that I may be getting something wrong, but I looked in the sample and I tried both notations (which I'm guessing both work) for the fonts variants and I just can't get the plan to work. It always builds the default Iosevka instead of the variants I cherry-picked.

---

* Your font version: (Release or git, version) v
* Your font variant: Custom
* Your operating system (name and version): MacOS Big Sur
* Your application using Iosevka: Doesn't apply

If you have problem when building Iosevka, please provide these information:

* Your Node.js version: v10.15.3
* Your _otfcc_ version: 0.9.6
* Your `private-build-plans.toml`, if present:
```
[buildPlans.iosevka-code]      # <iosevka-code> is your plan name
family = "Iosevka Code"        # Font menu family name
spacing = "normal"               # Optional; Values: `normal`, `term`, `fontconfig-mono`, or `fixed`
serifs = "sans"                  # Optional; Values: `sans` or `slab`

###################################################################################################
# Configure variants

# Optional; Whether to inherit a `ss##` variant
# [buildPlans.iosevka-code.variants]
# inherits = "ss01"

# Optional; Configure single character's variant
[buildPlans.iosevka-code.variants.design]
turn-v = 'curly'# cv01 = 2
capital-g = 'toothless-corner'# cv04 = 2
capital-k = 'curly'# cv07 = 2
capital-r = 'curly'# cv10 = 2
capital-y = 'curly'# cv11 = 2
b = 'toothless-corner'# cv13 = 2
j = 'flat-hook-serifed'# cv19 = 4
t = 'flat-hook'# cv27 = 3
v = 'curly'# cv29 = 2
w = 'curly'# cv30 = 2
x = 'curly'# cv31 = 2
y = 'curly'# cv32 = 3
z = 'curly'# cv33 = 4
zero = 'dotted'# cv36 = 2
one = 'base'# cv37 = 2
four = 'semi-open-non-crossing'# cv39 = 4
six = 'open-contour'# cv40 = 2
seven = 'serifed'# cv41 = 2
nine = 'open-contour'# cv42 = 2
asterisk = 'hexlow'# cv44 = 4
paren = 'large-contour'# cv48 = 2
brace = 'curly'# cv49 = 2
number-sign = 'slanted-open'# cv50 = 4
ampersand = 'upper-open'# cv51 = 2
dollar = 'open'# cv53 = 1
lig-ltgteq = 'slanted'# cv56 = 2



# Optional; Configure single characters variant for Upright and Oblique; Overrides [design]
[buildPlans.iosevka-code.variants.upright]
a = 'doublestorey-toothless-corner'# cv12 = 3
d = 'toothless-corner'# cv14 = 2
f = 'flat-hook'# cv15 = 9
g = 'earless-corner'# cv16 = 4
h = 'straight'# cv17 = 1
i = 'hooky'# cv18 = 3
k = 'curly'# cv20 = 2
l = 'serifed-flat-tailed'# cv21 = 10
m = 'earless-corner-double-arch'# cv22 = 5
n = 'earless-corner-straight'# cv23 = 3
p = 'earless-corner'# cv24 = 2
q = 'earless-corner'# cv25 = 3
r = 'earless-corner'# cv26 = 4
u = 'toothless-corner'# cv28 = 2

# Optional; Configure single characters variant for Italic only; Overrides [design]
[buildPlans.iosevka-code.variants.italic]
a = 'singlestorey-tailed'# cv12 = 6
d = 'toothed'# cv14 = 4
f = 'flat-hook-tailed'# cv15 = 11
g = 'opendoublestorey'# cv16 = 3
h = 'motion-serifed-tailed'# cv17 = 4
i = 'serifed-tailed'# cv18 = 8
k = 'cursive'# cv20 = 3
l = 'serifed-tailed'# cv21 = 8
m = 'motion-serifed-tailed'# cv22 = 19
n = 'motion-serifed-tailed'# cv23 = 8
p = 'motion-serifed'# cv24 = 4
q = 'tailed'# cv25 = 2
r = 'top-serifed'# cv26 = 3
u = 'motion-serifed-tailed'# cv28 = 6

# End variant section
###################################################################################################

###################################################################################################
# Configure ligations

[buildPlans.iosevka-code.ligations]
inherits = "clike"   # Optional; inherits an existing ligation set
disables = []               # Optional; disable specific ligation groups, overrides inherited ligation set
enables  = []               # Optional; enable specific ligation groups, overrides inherited ligation set

# End ligation section
###################################################################################################


###################################################################################################
# Override default building weights
# When buildPlans.<plan name>.weights is absent, all weights would built and mapped to
# default values.
# IMPORTANT : Currently "menu" and "css" property only support numbers between 0 and 1000.
#             and "shape" properly only supports number between 100 and 900 (inclusive).
#             If you decide to use custom weights you have to define all the weights you
#             plan to use otherwise they will not be built.
[buildPlans.iosevka-code.weights.light]
shape = 300  # Weight for glyph shapes.
menu  = 300  # Weight for the font's names.
css   = 300  # Weight for webfont CSS.

[buildPlans.iosevka-code.weights.regular]
shape = 400  # Weight for glyph shapes.
menu  = 400  # Weight for the font's names.
css   = 400  # Weight for webfont CSS.

[buildPlans.iosevka-code.weights.book]
shape = 450
menu  = 450  # Use 450 here to name the font's weight "Book"
css   = 450

[buildPlans.iosevka-code.weights.medium]
shape = 500
menu  = 500
css   = 500

[buildPlans.iosevka-code.weights.bold]
shape = 700
menu  = 700
css   = 700

[buildPlans.iosevka-code.weights.black]
shape = 900
menu  = 900
css   = 900

# End weight section
###################################################################################################

###################################################################################################
# Override default building slope sets
# Format: <upright|italic|oblique> = <"normal"|"italic"|"oblique">
# When this section is absent, all slopes would be built.

[buildPlans.iosevka-code.slopes]
upright = "normal"
italic = "italic"

# End slope section
###################################################################################################

###################################################################################################
# Override default building widths
# When buildPlans.<plan name>.widths is absent, all widths would built and mapped to
# default values.
# IMPORTANT : Currently "shape" property only supports numbers between 434 and 664 (inclusive),
#             while "menu" only supports integers between 1 and 9 (inclusive).
#             The "shape" parameter specifies the unit width, measured in 1/1000 em. The glyphs'
#             width are equal to, or a simple multiple of the unit width.
#             If you decide to use custom widths you have to define all the widths you plan to use,
#             otherwise they will not be built.

[buildPlans.iosevka-code.widths.normal]
shape = 576        # Unit Width, measured in 1/1000 em.
menu  = 7          # Width grade for the font's names.
css   = "expanded"   # "font-stretch' property of webfont CSS.

# [buildPlans.iosevka-code.widths.extended]
# shape = 576
# menu  = 7
# css   = "expanded"

# End width section
###################################################################################################

###################################################################################################
# Character Exclusion
# Specify character ranges in the section below to exclude certain characters from the font being
# built. Remove this section when this feature is not needed.

[buildPlans.iosevka-code.exclude-chars]
ranges = [[10003, 10008]]

# End character exclusion
###################################################################################################

###################################################################################################
# Compatibility Ligatures
# Certain applications like Emacs does not support proper programming liagtures provided by
# OpenType, but can support ligatures provided by PUA codepoints. Therefore you can edit the
# following section to build PUA characters that are generated from the OpenType ligatures.
# Remove this section when compatibility ligatures are not needed.

[[buildPlans.iosevka-code.compatibility-ligatures]]
unicode = 57600 # 0xE100
featureTag = 'calt'
sequence = '<*>'

# End compatibility ligatures section
###################################################################################################

###################################################################################################
# Metric overrides
# Certain metrics like line height (leading) could be overridden in your build plan file.
# Edit the values to change the metrics. Remove this section when overriding is not needed.

[buildPlans.iosevka-code.metric-override]
leading = 1250
winMetricAscenderPad = 0
winMetricDescenderPad = 0
powerlineScaleY = 1
powerlineScaleX = 1
powerlineShiftY = 0
powerlineShiftX = 0

# End metric override section
###################################################################################################
  ```

* The detailed error log: It's not failing, just building the build I don't expect

**Comment by be5invis** [comment]
Is this what you expect? You need to run `npm run build -- contents::iosevka-code` to get them built.

![image](https://user-images.githubusercontent.com/240091/100489678-979b5a80-30ca-11eb-9858-5ae9961d5b2b.png)

The config is like this

```toml

[buildPlans.iosevka-code]      # <iosevka-code> is your plan name
family = "Iosevka Code"        # Font menu family name
spacing = "normal"               # Optional; Values: `normal`, `term`, `fontconfig-mono`, or `fixed`
serifs = "sans"                  # Optional; Values: `sans` or `slab`

# Optional; Configure single character's variant
[buildPlans.iosevka-code.variants.design]
turn-v = 'curly'# cv01 = 2
capital-g = 'toothless-corner'# cv04 = 2
capital-k = 'curly'# cv07 = 2
capital-r = 'curly'# cv10 = 2
capital-y = 'curly'# cv11 = 2
b = 'toothless-corner'# cv13 = 2
j = 'flat-hook-serifed'# cv19 = 4
t = 'flat-hook'# cv27 = 3
v = 'curly'# cv29 = 2
w = 'curly'# cv30 = 2
x = 'curly'# cv31 = 2
y = 'curly'# cv32 = 3
z = 'curly'# cv33 = 4
zero = 'dotted'# cv36 = 2
one = 'base'# cv37 = 2
four = 'semi-open-non-crossing'# cv39 = 4
six = 'open-contour'# cv40 = 2
seven = 'serifed'# cv41 = 2
nine = 'open-contour'# cv42 = 2
asterisk = 'hexlow'# cv44 = 4
paren = 'large-contour'# cv48 = 2
brace = 'curly'# cv49 = 2
number-sign = 'slanted-open'# cv50 = 4
ampersand = 'upper-open'# cv51 = 2
dollar = 'open'# cv53 = 1
lig-ltgteq = 'slanted'# cv56 = 2

# Optional; Configure single characters variant for Upright and Oblique; Overrides [design]
[buildPlans.iosevka-code.variants.upright]
a = 'doublestorey-toothless-corner'# cv12 = 3
d = 'toothless-corner'# cv14 = 2
f = 'flat-hook'# cv15 = 9
g = 'earless-corner'# cv16 = 4
h = 'straight'# cv17 = 1
i = 'hooky'# cv18 = 3
k = 'curly'# cv20 = 2
l = 'serifed-flat-tailed'# cv21 = 10
m = 'earless-corner-double-arch'# cv22 = 5
n = 'earless-corner-straight'# cv23 = 3
p = 'earless-corner'# cv24 = 2
q = 'earless-corner'# cv25 = 3
r = 'earless-corner'# cv26 = 4
u = 'toothless-corner'# cv28 = 2

# Optional; Configure single characters variant for Italic only; Overrides [design]
[buildPlans.iosevka-code.variants.italic]
a = 'singlestorey-tailed'# cv12 = 6
d = 'toothed'# cv14 = 4
f = 'flat-hook-tailed'# cv15 = 11
g = 'opendoublestorey'# cv16 = 3
h = 'motion-serifed-tailed'# cv17 = 4
i = 'serifed-tailed'# cv18 = 8
k = 'cursive'# cv20 = 3
l = 'serifed-tailed'# cv21 = 8
m = 'motion-serifed-tailed'# cv22 = 19
n = 'motion-serifed-tailed'# cv23 = 8
p = 'motion-serifed'# cv24 = 4
q = 'tailed'# cv25 = 2
r = 'top-serifed'# cv26 = 3
u = 'motion-serifed-tailed'# cv28 = 6

[buildPlans.iosevka-code.ligations]
inherits = "clike"

[buildPlans.iosevka-code.weights.light]
shape = 300  # Weight for glyph shapes.
menu  = 300  # Weight for the font's names.
css   = 300  # Weight for webfont CSS.

[buildPlans.iosevka-code.weights.regular]
shape = 400  # Weight for glyph shapes.
menu  = 400  # Weight for the font's names.
css   = 400  # Weight for webfont CSS.

[buildPlans.iosevka-code.weights.book]
shape = 450
menu  = 450  # Use 450 here to name the font's weight "Book"
css   = 450

[buildPlans.iosevka-code.weights.medium]
shape = 500
menu  = 500
css   = 500

[buildPlans.iosevka-code.weights.bold]
shape = 700
menu  = 700
css   = 700

[buildPlans.iosevka-code.weights.black]
shape = 900
menu  = 900
css   = 900

[buildPlans.iosevka-code.slopes]
upright = "normal"
italic = "italic"

[buildPlans.iosevka-code.widths.normal]
shape = 576        # Unit Width, measured in 1/1000 em.
menu  = 7          # Width grade for the font's names.
css   = "expanded"   # "font-stretch' property of webfont CSS.

```

**Comment by arudp** [comment]
That's what I thought, I was running the just ttf and I thought maybe that was it so I just tried with the config as you wrote it with here to no avail. The output is still the default build. I'm gonna tell you the process I followed just in case to see if you spot something I did wrong:

```
git pull
npm install
code private-build-plans.toml # Insert the config I pasted and then the one you pasted
npm run build -- contents::iosevka-code # First tried with ttf::iosevka-code
```

I'm going to delete my copy of the repo and do a fresh clone to see if there's something wrong with that, maybe.

Thank you like always, Belleve, by the way, for the amazing font and your awesome assistance!
